components/CookieConsentConfig.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @type {UserConfig}
+ */
+const pluginConfig = {
+  current_lang: "en",
+  autoclear_cookies: true, // default: false
+  page_scripts: true, // default: false
+  test: "ciao",
+  force_consent: true, // default: false
+  // mode: 'opt-in'                          // default: 'opt-in'; value: 'opt-in' or 'opt-out'
+  // delay: 0,                               // default: 0
+  // auto_language: null                     // default: null; could also be 'browser' or 'document'
+  // autorun: true,                          // default: true
+  // hide_from_bots: false,                  // default: false
+  // remove_cookie_tables: false             // default: false
+  // cookie_name: 'cc_cookie',               // default: 'cc_cookie'
+  // cookie_expiration: 182,                 // default: 182 (days)
+  // cookie_necessary_only_expiration: 182   // default: disabled
+  // cookie_domain: location.hostname,       // default: current domain
+  // cookie_path: '/',                       // default: root
+  // cookie_same_site: 'Lax',                // default: 'Lax'
+  // use_rfc_cookie: false,                  // default: false
+  // revision: 0,                            // default: 0
+
+  onFirstAction: function (user_preferences, cookie) {
+    // callback triggered only once
+  },
+
+  onAccept: function (cookie) {
+    // ...
```

Comment:
Any documentation on what this function do! As right now I am not sure it is handling any action(s).

Reply:
![image](https://user-images.githubusercontent.com/76250685/212443551-9098590c-a8a0-4079-aa8a-8bb1a4388d3f.png)

This function isn't handling any action as of now, this function is for loading extra scripts.